### ISSUE TITLE
quickMove: Parent folder with space in name not shown

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#297]
If a parent folder in a search string (with a `/` or `>` character contained in it) has a space in the name it is not listed in the search box of quickMove:

it is shown here as a "regular" result:

![image](https://user-images.githubusercontent.com/6314674/184325794-15075e3a-6963-4d8a-9a61-b73f5c320d52.png)

If the first word is incomplete the match fails "no matching parent folders":

![image](https://user-images.githubusercontent.com/6314674/184326011-64496d23-b55e-4eba-ae2d-8a988859a7dc.png)

If the starting string is entered _without_ abbreviating the first word, including the space - there is no "create subfolder" command offered:

![image](https://user-images.githubusercontent.com/6314674/184326179-a3531137-9c64-42df-bdab-17ac971ba7b5.png)

**Comment by RealRaven2000** [comment]
For clarity I changed the label that is displayed when a folder matches but has no child folders:

![image](https://user-images.githubusercontent.com/6314674/184510931-5c33082b-8d4c-4588-812f-c335b4a01e5b.png)

Test version

[QuickFolders-mx-5.12pre43.zip](https://github.com/RealRaven2000/QuickFolders/files/9332185/QuickFolders-mx-5.12pre43.zip)

To install this preview: download the zip file. DON'T EXTRACT it. Drag the zip file into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
Added the other delimiter characters that we supported in the target folder search as well: `-` `_` `@` `+` `&` `.` and Space. To match any of these use "space" to delimit the words in the search text. For instance to find the parten folder "John&Martha" you just enter "j m/" or "jo ma/" - just make sure to enter the first letter(s) of each part with a space between each part.

[QuickFolders-mx-5.12pre46.zip](https://github.com/RealRaven2000/QuickFolders/files/9334184/QuickFolders-mx-5.12pre46.zip)

To install this preview: download the zip file. DON'T EXTRACT it. Drag the zip file into Thunderbird Add-ons Manager.
